FirebaseAuth/Tests/Unit/PhoneAuthProviderTests.swift

```diff
@@ -575,9 +571,21 @@
           }
         }
       }
-
+      if reCAPTCHAfallback {
```

Comment:
where did we set the rcE status to be OFF in these tests?

Reply:
rCE is off by default in rpcIssuer since we didn't want to break any other tests